fake_property_data.py

```diff
@@ -0,0 +1,135 @@
+            distance_from_airport = random.randint(1, 20),
+            built_in_year=random.randint(1900, timezone.now().year), 
+            kitchen = random.randint(1, 3),
+            floors = random.randint(1, 10),
+            maid_room = fake.boolean(),
+            built_in_wardrobes = fake.boolean(),
+            kitchen_appliances = fake.boolean(),
+            lower_portion = fake.boolean(),
+            Farmhouse = fake.boolean(),
+            electricity_backup = fake.boolean(),
+            furnished_unfurnished = fake.boolean(),
+            covered_parking = fake.boolean(),
+            lobby_in_building = fake.boolean(),
+            security = fake.boolean(),
+            parking_space = fake.boolean(),
+            drawing_room = fake.boolean(),
+            study_room = fake.boolean(),
+            laundry_room = fake.boolean(),
+            store_room = fake.boolean(),
+            lounge_sitting_area = fake.boolean(),
+            internet = fake.boolean(),
+            mosque = fake.boolean(),
+            kids_play_area = fake.boolean(),
+            medical_center = fake.boolean(),
+            community_lawn_garden = fake.boolean(),
+            near_by_school = fake.boolean(),
+            near_by_hospital = fake.boolean(),
+            near_by_shopping_mall =fake.boolean(), 
+            other_description = fake.sentence(),
+            # Add other amenities fields as needed
+        )
+        property_type = PropertyTypes.objects.create(
+            home_types=random.choice(['house', 'flat', 'room', 'pent_house']),
+            # Add other property types data as needed
+        )
+        property_location = PropertyLocation.objects.create(
+            latitude=fake.latitude(),
+            longitude=fake.longitude()
+        )
+        installment = PropertyInstallment.objects.create(
+            advance_amount=random.randint(10000, 50000),
+            no_of_inst=random.randint(6, 24),
+            monthly_inst=random.randint(5000, 10000),
+            ready_for_possession=fake.boolean()
+        )
+
+        # Create Property instance
+        property_instance = Property.objects.create(
+            title=fake.sentence(),
+            phone=fake.phone_number(),
+            landline=fake.phone_number(),
+            secondry_phone=fake.phone_number(),
+            email=fake.email(),
+            rent_sale_type=random.choice(['rent', 'sale']),
+            area=area,
+            agent=agent,
+            amenties=amenities,
+            property_type=property_type,
+            property_location=property_location,
+            installment=installment,
+            available=fake.boolean(),
+            description=fake.paragraph(),
+            total_price=random.randint(100000, 1000000),
+            date=fake.date_between(start_date='-1y', end_date='today')
+        )
+
+        # Create Media instances associated with the property
+        for _ in range(random.randint(1, 5)):
```

Comment:
`random.randint(5, 10)`
`width: random.randint(400, 600)`
`heigth: random.randint(600, 800)`

Reply:
The fake.image_url() method in the Faker library does not directly allow you to set specific width and height